Library/Models/Survey.cs

```diff
@@ -65,5 +65,10 @@ public Survey(SurveyType surveyType)
         /// True if the survey is a blueprint survey, otherwise false
         /// </summary>
         public bool IsBlueprint { get; set; } = false;
+
+        /// <summary>
+        /// True if the survey can be managed by the Respondents Gateway
+        /// </summary>
+        public bool EnableRespondentsGateway { get; set; } = false;
```

Comment:
I'm ok with explicitly initialize the property to false, but should not be needed as it's the default value for bool

Reply:
Yeah. I just left it because IsBlueprint was doing the same (to be consistent). I could have removed both actually.